lib/pharos/cluster_manager.rb

```diff
@@ -47,12 +47,16 @@ def validate
     def apply_phases
       apply_phase(Phases::ValidateHost, config.hosts, ssh: true, parallel: true)
       apply_phase(Phases::ConfigureHost, config.hosts, ssh: true, parallel: true)
+      apply_phase(Phases::ConfigureCfssl, config.etcd_hosts, ssh: true, parallel: true)
+      apply_phase(Phases::ConfigureEtcdCa, [config.etcd_host], ssh: true, parallel: false)
+      apply_phase(Phases::ConfigureEtcd, config.etcd_hosts, ssh: true, parallel: true)
 
-      apply_phase(Phases::ConfigureMaster, config.master_hosts, ssh: true, parallel: false)
+      apply_phase(Phases::ConfigureMaster, config.master_hosts, ssh: true, parallel: false, master: config.master_host)
```

Comment:
The `master: ...` here is problematic, because you can't actually use it for `Pharos::Kube.client` until the `Phases::ConfigureClient` has been run. That's why I left it out of all these earlier phases... we shouldn't need it for the upgrade case anymore?

Reply:
Yeah, it seems that `ConfigureMaster` is not using master anymore.. will cleanup.